smartsim/experiment.py

```diff
@@ -44,6 +44,9 @@
 
 logger = get_logger(__name__)
 
+_OutputFormatString = t.Optional[t.Literal["html"]]
+_VerbosityLevelString = t.Literal["info", "debug", "developer"]
```

Comment:
- _VerbosityLevelString is redefined in experiment after definition in `previewrenderer.py`
- _OutputFormatString is redefined in experiment after definition in `previewrenderer.py`

Reply:
removed